gcp/shell_startup.sh

```diff
@@ -20,4 +20,4 @@ if [ ! -f "$HOME/bin/terraform" ]; then
 fi
 
 echo ""
-echo "Your shell is almost ready, please restart your terminal before running any Lacework CLI command!"
+echo "Your shell is almost ready. Type `exit` then hit enter before running any Lacework CLI command. Open the Cloud Shell again and the Lacework CLI will be ready for use!"
```

Comment:
Could you verify that this command actually prints out "exit" and not execute it. 🤔 I have a feeling that it might not, if it looks good then, this change looks good to me!

Reply:
@scottford-lw I just verified (sorry for the delay) and this is the result of running that `echo` command.
```
➜  ~ echo "Your shell is almost ready. Type `exit` then hit enter before running any Lacework CLI command. Open the Cloud Shell again and the Lacework CLI will be ready for use"
Your shell is almost ready. Type  then hit enter before running any Lacework CLI command. Open the Cloud Shell again and the Lacework CLI will be ready for use
```

Could you please update the ` in both shell scripts for Azure and GCP?